src/js/images.js

```diff
@@ -426,6 +427,28 @@
         return data.context;
     };
 
+
+    /**
+     * Generate a unique numeral hash code based on a string
+     *
+     * @param {string} str
+     * @return {string}
+     */
+
+   Images.prototype.hashCode = function(str) {
```

Comment:
Don't need to add the method to the prototype. It can be a simple method anywhere in the file(or where invoked) which need not be accessible outside.

@hchhabra _a